packages/grpc-js/src/compression-filter.ts

```diff
@@ -138,7 +138,9 @@ class UnknownHandler extends CompressionHandler {
   compressMessage(message: Buffer): Promise<Buffer> {
     return Promise.reject<Buffer>(
       new Error(
-        `Received message compressed wth unsupported compression method ${this.compressionName}`
+        `Received message compressed wth unsupported compression method ${
```

Comment:
That's a bit of a weird looking change. What's the reasoning?

Reply:
I think this came from the code formatter.

Reply:
There’s a typo in `wth` — it should be `with`.